### ISSUE TITLE
Support Fairpool

### ISSUE THREAD

**Issue by abenhamo** [rodneymo/rig-monitorv2#11]
could you please add support to fairpool : 
here the api:  https://loki.fairpool.xyz/api/stats?login= WALLET HERE

**Comment by rodneymo** [comment]
I am looking into this now. I could find some info about the URLs.
Could you provide me a json example?
Also for the network API: https://loki.fairpool.xyz/api/network

**Comment by abenhamo** [comment]
here we go: {"unconfirmed": 5264637495.0, "invalid": false, "threshold": 5000000000, "workers": [[null, 63597.5]], "balance": 3575241503.0, "method": "stats", "paid": 2401903571070.0, "blocksTillNextConfirmation": 1}

**Comment by rodneymo** [comment]
There's a null value in the workers array. Any ideas what data is going there?

**Comment by abenhamo** [comment]
it's the worker name

**Comment by xiphon** [comment]
Another example:
```json
{
    "paid": 121707944680723.0,
    "balance": 0.0,
    "method": "stats",
    "workers": [
        [
            null,
            7181.12
        ],
        [
            "Worker1",
            0.0
        ],
        [
            "Worker2",
            null
        ],
        [
            "Worker3",
            5013.62
        ]
    ],
    "invalid": false,
    "unconfirmed": 0.0,
    "threshold": 5000000000000,
    "blocksTillNextConfirmation": null
}
```

Worker name:
`null` - Worker name is not specified, default
`string` - Worker name

Worker state based on the speed field (2nd value in the array):
`null` - Worker is offline
`float` - Worker is online

Let me know if you have any questions.

**Comment by rodneymo** [comment]
Thanks.
Support has been added to development version 2.2.d1 (see releases). A couple of comments:
-I calculated the total hr and stored it the DB
-I could not do E2E testing as I don't have a valid wallet (and don't want to use my live rigs for this)
-I couldn't tell the units used for paid, balance and unconfirmed so I just stored the figure as a float. You'll need to adjust the units in order to make it readable in Grafana e.g. for paid, balance and unconfirmed graphs I divided by 1E9.
If you have any problems please create an issue and attach a trace. Info about running traces is here: http://randomcryptostuff.blogspot.nl/2018/05/customizing-and-troubleshooting-rig.html

Final comment. This will be included in the 2.2 release in 1-2 weeks time.
